pkg/allocation/converters/converter_test.go

```diff
@@ -1341,7 +1354,7 @@ func TestConvertGSAToAllocationResponse(t *testing.T) {
 			defer runtime.FeatureTestMutex.Unlock()
 			require.NoError(t, runtime.ParseFeatures(tc.features))
 
```

Comment:
I would just default here so the existing tests work (and add one more test that something other than the default also works). So like:

```
grpcUnallocatedStatusCode := tc.grpcUnallocatedStatusCode
if grpcUnallocatedStatusCode == codes.OK {
  grpcUnallocatedStatusCode = codes.ResourceExhausted
}

out, err := Convert(tc.in, grpcUnallocatedStatusCode)
```